composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "juampi92/phecks",
+  "name": "studocu/phecks",
```

Comment:
So sad 😢

Reply:
https://github.com/juampi92/phecks/pull/1/commits/a7b75ec9a7bf7e17934be2cb1aa6b005be18c2c6 I reverted this because I'm merging to the original package